interview-questions.md

```diff
@@ -7,60 +7,73 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer:  git is the local control version on the local computer that can be edited and such, then the difference between git and github is that github is on the remote web.
 
-  Researched answer:
+  Researched answer:  git is a modern version control , that basically means it tracks changes and can control who adds code to the main, I think it is called the 'source of truth', part of the code. And Github is a cloud based system that allows storage and distribution of your git repositories, just an easier place to manage your git
 
 
 
 2. What types of evaluations will return a Boolean value?
 
   Your answer:
+  comparisons and booleans, the only things that I can really think of are things that would be in if statements, and those are compared to determine whether they are true or false, and those comparisons are things like ==, ===, >. <. true, false, !
```

Comment:
Yup, you got it. There are built in methods that can return boolean values as well.